### PR TITLE
OWNERS: Introduce dep-approvers

### PR DESCRIPTION
**Pull request by fabiand**
Introduce a dedicated group for approving dependency changes Primary objective of this group of people is to ensure that the number of dependencies is kept small.

### What this PR does
Before this PR:
approvers could approve deps

After this PR:
Dedicated group is required to approve deps

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note

```

### CODE REVIEW

**Comment by fabiand** [comment]
@rthallisey @rmohr @davidvossel fyi for having a second set of eyes on dependencies

cc @alicefr @alicefr @xpivarc

**Comment by xpivarc** [comment]
@fabiand you mean @alicefr ?

**Review by EdDev — COMMENTED** [review]
I am in favor of this approach.

Can we establish the policy or agreement by which the group will follow?
Do we have examples of other projects following something similar?

As the members may have conflict of interest when reviewing/approving PRs with dep changes, some basic rules and scope definitions can assist with the role.

**Comment by fabiand** [comment]
@xpivarc I mean @alicefr :D Thanks!

**Comment by fabiand** [comment]
@EdDev +1

I'd hope that the small group of three @xpivarc @alicefr @fossedihelm could think of 1-3 simple rules to provide the guardrails. Let's keep it simple.

**Comment by lyarwood** [comment]
> Introduce a dedicated group for approving dependency changes Primary objective of this group of people is to ensure that the number of dependencies is kept small.

Silly question but what's driving this? Have our dependencies grown to an unmanageable level?

**Comment by fabiand** [comment]
Yes, to keep it manageable, and just see how existing dependencies can be reused instead of adding new ones.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -157,3 +157,17 @@ aliases:
...
   sig-architecture-approvers: []
   sig-architecture-reviewers: []
+
+  #
+  # Dependency Approvers
+  # Responsibility of this group is to keep KubeVirt's dependencies
+  # small.
+  #
+  dep-approvers:
+    - afrosi
```
Comment (alicefr) [inline OWNERS_ALIASES]: alicefr :)

**Comment by alicefr** [comment]
@fabiand do you want us to write the rules here or in a follow-up PR?

**Comment by alicefr** [comment]
Additionally, I think before engaging, I'd like to understand the scope of the sig. If this means, we are responsible for all the dependencies, I feel its a huge responsibility. In general, I think the load should it spread between all maintainers and part of the review process.

**Pull request closed without merging**